UI/GameSettingsScreen.cpp

```diff
@@ -338,6 +342,10 @@ void GameSettingsScreen::CreateViews() {
 	texSecondary_->SetDisabledPtr(&g_Config.bSoftwareRendering);
 
 	CheckBox *framebufferSlowEffects = graphicsSettings->Add(new CheckBox(&g_Config.bDisableSlowFramebufEffects, gr->T("Disable slower effects (speedup)")));
+	framebufferSlowEffects->OnClick.Add([=](EventParams &e) {
+		settingInfo_->Show(gr->T("DisableSlowerEffect Tip", "Improvement performance in some scenarios on old device"));
```

Comment:
I don't think this is a good description of what it does.  Maybe, "Make special effects look bad to improve performance".  Hopefully this is already obvious by the name?

-[Unknown]